### ISSUE TITLE
Clinical modifier: Worse in evening

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4736]
**Preferred term label:**
worse in evening

**Synonyms**

**Definition (free text, please give PubMed ID)**
Applies to a sign or symptom that is exacerbated in the evening as compared to the day.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
